src/datasources/asset-calibration/__snapshots__/AssetCalibrationDataSource.test.ts.snap

```diff
@@ -121,7 +121,7 @@ exports[`queries calibration forecast with location groupBy 1`] = `
       {
         "name": "Group",
         "values": [
-          "Location1",
+          "my system",
```

Comment:
Please give it a proper name Like Minion1-alias

Reply:
this values are used in the system test. I will change it Minion1-alias